vue-app/src/components/BrightIdWidget.vue

```diff
@@ -1,21 +1,20 @@
 <template>
   <div v-if="isProjectCard" class="setup-container-project-card">
```

Comment:
This code is not very DRY - most of this is entirely re-used when `isProjectCard` is `false`. Let's refactor this a bit.

Reply:
See what I mean?
![Image 2021-08-09 at 6 02 21 PM](https://user-images.githubusercontent.com/8097623/128793020-6f21d3af-8405-4cdb-81c2-9b0b1f92fac5.jpg)
Seems the only difference is when this widget is in the Profile (i.e. `!isProjectCard`), it also has that lower row (highlighted in red)

Reply:
Good point, will take a towel to this and DRY it up